### ISSUE TITLE
Elaborate on how to use this with Java 17

### ISSUE THREAD

**Issue by charlesritchea** [openpnp/opencv#79]
Now that loadShared is essentially deprecated, and I assume many people using that method, like myself, have no idea how to get the exact version of opencv needed built in their Dockerfile to be used by loadLocally. Can we get an example Dockerfile or any further instructions on how to setup an environment that works with loadLocally. Maybe I'm dumb, but I don't understand what is needed from just the current README

**Comment by fighttiger25** [comment]
Hi, @charlesritchea , did you solve the problem in the end? I encountered the same problem these days.

**Comment by charlesritchea** [comment]
Yeah it ended up being stupidly simple I think it's just to call loadLocally, I'll put the code here when I get back

**Comment by charlesritchea** [comment]
@fighttiger25 it was just this:
```
    OpenCV.loadLocally()
```

**Comment by charlesritchea** [comment]
I thought I had to load the specific native library, but this just does it automatically from the library bundled in the maven dependency

**Comment by fighttiger25** [comment]
Thanks for your quick response! May I ask you about the version of JRE and OpenCV you used?
My JRE crashed after successfully loading the library `OpenCV.loadLocally()` and printing out the OpenCV version.
The log is as follows:


```
OpenCV 4.5.5
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x000000000020e936, pid=1, tid=135
#
# JRE version: OpenJDK Runtime Environment Temurin-17.0.4.1+1 (17.0.4.1+1) (build 17.0.4.1+1)
# Java VM: OpenJDK 64-Bit Server VM Temurin-17.0.4.1+1 (17.0.4.1+1, mixed mode, sharing, tiered, compressed oops, compressed class ptrs, g1 gc, linux-amd64)
 # Problematic frame:
 # C  0x000000000020e936`
```
